Formula/kcov.rb

```diff
@@ -0,0 +1,31 @@
+class Kcov < Formula
+  desc "Code coverage tool for compiled programs, Python and Bash"
+  homepage "https://simonkagstrom.github.com/kcov/index.html"
+  url "https://github.com/SimonKagstrom/kcov/archive/v35.tar.gz"
+  sha256 "74c172dae2ac2866e0adc91d3fd80276e5acb970d11ac71679a0f7336897a476"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", *std_cmake_args
+    system "make"
+
+    # As per notes in https://github.com/SimonKagstrom/kcov/issues/166
+    MachO::Tools.change_install_name(
+      "src/kcov",
+      "@rpath/LLDB.framework/LLDB",
+      "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/LLDB",
```

Comment:
It's not particularly desirable, but until https://github.com/SimonKagstrom/kcov/issues/166 is fixed it's needed to make it work. I've added Xcode as an explicit dependency for this tool (which shouldn't be a big deal, as most people needing a code coverage tool are also compiling stuff)

Reply:
This seems a bit concerning given that may not even be the path to Xcode on the user's machine even when `depends_on :xcode` is satisfied.

Reply:
I have just replaced the hardcoded path with one using the `Xcode` module

Reply:
Unfortunately, that still doesn't resolve the issue since the `install` method will only run during bottling, which means the path will be correct for CI and people building from source but not necessarily for people using the bottles.

Reply:
I've just added a `pour_bottle?` block to explicitly always build from source